python/tvm/relay/backend/vm.py

```diff
@@ -131,9 +131,11 @@ def __init__(self, mod, ctx, target):
         self.ctx = ctx
         self.target = target
 
-    def _make_executor(self, expr):
-        assert isinstance(expr, Expr)
-        self.mod[self.mod.entry_func] = expr
+    def _make_executor(self, expr=None):
+        expr = expr if expr else self.mod
```

Comment:
I think we should try to factor all this normalization code into one spot.

Reply:
IMO we should only take module and remove expr related code

Reply:
I also think we probably only should just take a module, but it looks we need to refactor the interpreter and the graph_runtime somehow.